docs/DISTROS.md

```diff
@@ -0,0 +1,170 @@
+
+## Arch Linux
+
+- curl: https://archlinux.org/packages/core/x86_64/curl/
+- curl issues: https://gitlab.archlinux.org/archlinux/packaging/packages/curl/-/issues
+- curl security: https://security.archlinux.org/package/curl
+- curl wiki: https://wiki.archlinux.org/title/CURL
+
+# Clear Linux
+
+- curl: https://github.com/clearlinux-pkgs/curl
+- curl issues: https://github.com/clearlinux/distribution/issues
+
+# Conary
+
+- curl: https://github.com/conan-io/conan-center-index/tree/master/recipes/libcurl
+- curl issues: https://github.com/conan-io/conan-center-index/issues
+- curl patches: https://github.com/conan-io/conan-center-index/tree/master/recipes/libcurl (in `all/patches/*`, if any)
+
+# conda-forge
+
+- curl: https://github.com/conda-forge/curl-feedstock
+- curl issues: https://github.com/conda-forge/curl-feedstock/issues
+
+## curl-for-win
+
+(this is the official curl binaries for Windows shipped by the curl project)
+
+- curl: https://curl.se/windows/
+
+Issues and patches for this are managed in the main curl project.
+
+## Debian
+
+- curl: https://tracker.debian.org/pkg/curl
+- curl issues: https://bugs.debian.org/cgi-bin/pkgreport.cgi?pkg=curl
+- curl patches: https://udd.debian.org/patches.cgi?src=curl
+
+## Fedora
+
+- curl: https://src.fedoraproject.org/rpms/curl
+- curl issues: [bugzilla](https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&classification=Fedora&product=Fedora&product=Fedora%20EPEL&component=curl)
+- curl patches: [list of patches in package git](https://src.fedoraproject.org/rpms/curl/tree/rawhide)
+
+## Gentoo Linux
+
+- curl: https://packages.gentoo.org/packages/net-misc/curl
+- curl issues: https://bugs.gentoo.org/buglist.cgi?quicksearch=net-misc/curl
+- curl package sources and patches: https://gitweb.gentoo.org/repo/gentoo.git/tree/net-misc/curl/
+
+## GNU Guix
+
+- curl: https://git.savannah.gnu.org/gitweb/?p=guix.git;a=blob;f=gnu/packages/curl.scm;hb=HEAD
+- curl issues: https://issues.guix.gnu.org/search?query=curl
+
+## Homebrew
+
+- curl: https://formulae.brew.sh/formula/curl
+
+Homebrew's policy is that all patches and issues should be submitted upstream
+unless it is very specific to Homebrew's way of packaging software.
+
+## MacPorts
+
+- curl: https://ports.macports.org/port/curl
+- curl issues: https://trac.macports.org/query?status=!closed&port=~curl&desc=1&order=id
+- curl patches: https://github.com/macports/macports-ports/tree/master/net/curl/files
+
+## Mageia
+
+curl: https://svnweb.mageia.org/packages/cauldron/curl/current/SPECS/curl.spec?view=markup
+curl issues: https://bugs.mageia.org/buglist.cgi?bug_status=NEW&bug_status=UNCONFIRMED&bug_status=NEEDINFO&bug_status=UPSTREAM&bug_status=ASSIGNED&component=RPM%20Packages&f1=cf_rpmpkg&list_id=176576&o1=casesubstring&product=Mageia&query_format=advanced&v1=curl
+curl patches: https://svnweb.mageia.org/packages/cauldron/curl/current/SOURCES/
+
```

Comment:
Another line to add:

- curl patches in stable distro releases: https://svnweb.mageia.org/packages/updates/<STABLE_VERSION>/curl/current/SOURCES/

I don't see how to point people to the right place without making the version number a placeholder. GitHub's Markdown parser linkifies this literal text in a way that results in a link that lists the version options available, so this is probably fine like this. The stable release are probably going to be the more interesting to other people than the rolling release ones above since that's where backported patches are going to end up.